client/question/domain/FeedbackObjectFactory.js

```diff
@@ -186,6 +186,24 @@ tie.factory('FeedbackObjectFactory', [
       }
     };
 
+
+    /**
+     * Returns all of the feedback provided as a JSON dict, preserving type
+     * and content information.
+     *
+     * @returns [{string, string}] The feedback paragraphs list of dicts.
+     */
+    Feedback.prototype.getParagraphsAsListOfDicts = function() {
+      var feedbackParagraphs = [];
```

Comment:
(Optional) A bit cleaner would be:

    return this._paragraphs.map(function(paragraph) {
      return paragraph.toDict();
    });

toDict() should be defined on the paragraph object prototype, btw, rather than on the class. Might be worth changing it if it's not the case.